app/views/articles/_form.html.erb

```diff
@@ -21,6 +30,6 @@
   </div>
 
   <div>
-    <%= form.submit %>
+    <%= form.submit "Submit"%>
```

Comment:
Here is the great place to become familiar with `I18n`. Read about Rails Internationalization (I18n) API and use it here.